### PR TITLE
update and replace #3918

### PR DESCRIPTION
**Pull request by wlammen**
1. Fix the merge conflict
2. remove modification in AV's mathbox.  This is submitted separately.

Since #3918 received several approvals, I will merge this replacement and update soon after it is approved the first time.

### CODE REVIEW

**Review by benjub — APPROVED** [review]
Thank you Wolf !

File: set.mm [hunk 1]
```diff
@@ -42487,6 +42463,74 @@ Equality axioms (and Extensionality) yield
...
+
+  ${
+    $d x A $.  $d x B $.
+    $( Alternate proof of ~ pweq directly from the definition.  (Contributed by
+       NM, 21-Jun-1993.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    pweqALT $p |- ( A = B -> ~P A = ~P B ) $=
+      ( vx wceq cv wss cab cpw sseq2 abbidv df-pw 3eqtr4g ) ABDZCEZAFZCGNBFZCGA
+      HBHMOPCABNIJCAKCBKL $.
+  $}
```
Comment (avekens) [inline set.mm:42509]: The proof is exactly the same as for pweq!?

Comment (benjub) [reply set.mm:42509]: Good catch. Somehow, the new proof of `pweq` was not copied. I'll update it... and I'll also check the other changes I made. FYI:
```
$ git show e3b0b8dc5355964a30c3468353757e1e4733a3fd
[...]
+  $( Equality theorem for power class.  (Contributed by NM, 21-Jun-1993.)
+     (Proof shortened by BJ, 13-Apr-2024.) $)
+  pweq $p |- ( A = B -> ~P A = ~P B ) $=
+    ( wceq cpw wss eqimss sspw syl eqimss2 eqssd ) ABCZADZBDZKABELMEABFABGHKBAE
+    MLEBAIBAGHJ $.
+
+  ${
+    $d x A $.  $d x B $.
+    $( Alternate proof of ~ pweq directly from the definition.  (Contributed by
+       NM, 21-Jun-1993.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    pweqALT $p |- ( A = B -> ~P A = ~P B ) $=
+      ( vx wceq cv wss cab cpw sseq2 abbidv df-pw 3eqtr4g ) ABDZCEZAFZCGNBFZCGA
+      HBHMOPCABNIJCAKCBKL $.
```

Comment (benjub) [reply set.mm:42509]: Proof from #3918 restored in #3944.

**Pull request merged**